packages/tokamak/sdk/tasks/deposit-withdraw-ton.ts

```diff
@@ -0,0 +1,205 @@
+import { task, types } from 'hardhat/config'
+import '@nomiclabs/hardhat-ethers'
+import { predeploys } from '@eth-optimism/core-utils'
+import { BytesLike, ethers } from 'ethers'
+
+import {
+  CrossChainMessenger,
+  MessageStatus,
+  TONBridgeAdapter,
+  AddressLike,
+} from '../src'
+
+console.log('Setup task...')
+
+const privateKey = process.env.PRIVATE_KEY
+const TON = process.env.TON as AddressLike
```

Comment:
Cast `TON` as a string
```suggestion
const TON = process.env.TON || ""
```

Because `new ethers.Contract(addressOrName string...)` accepts the first param is string